libs/SerialNumberKit/include/SerialNumberKit.h

```diff
@@ -5,21 +5,28 @@
 #pragma once
 
 #include <array>
+#include <filesystem>
 #include <interface/drivers/MCU.h>
+#include <span>
 
 namespace leka {
 
 class SerialNumberKit
 {
-	static constexpr size_t SN_SIZE		  = 33;
-	static constexpr size_t SHORT_SN_SIZE = 16 + 1;
+	static constexpr size_t SN_SIZE			= 31 + 1;
+	static constexpr size_t SHORT_SN_SIZE	= 15 + 1;
+	static constexpr auto date_of_test_path = "fs/sys/date_of_test";
 
   public:
 	explicit SerialNumberKit(interface::MCU &mcu) : _mcu(mcu) {}
 
 	auto getSerialNumber() -> std::array<uint8_t, SN_SIZE> &;
 	auto getShortSerialNumber() -> std::array<uint8_t, SHORT_SN_SIZE> &;
```

Comment:
those functions can/must return an `std::span` instead of a reference to std::array. It will work exactly the same.

Reply:
Applied ✔️